.circleci/config.yml

```diff
@@ -125,12 +124,20 @@ jobs:
           name: Push Docker image to Quay
           command: |
             docker login -u $DOCKER_USER -p $DOCKER_PASS quay.io
-            docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_BRANCH
             docker push quay.io/plotly/image-exporter:$CIRCLE_SHA1
+
+            # tag and push as branch name
+            docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_BRANCH
             docker push quay.io/plotly/image-exporter:$CIRCLE_BRANCH
-            if [[ ! -z "${CIRCLE_TAG}" ]]; then
-              docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_TAG
-              docker push quay.io/plotly/image-exporter:$CIRCLE_TAG
+
+            # tag and push as version number
+            # inspired by: https://discuss.circleci.com/t/git-tag-deploys-in-2-0/9493/4
+            if git log -1 --pretty=%B | grep "v^[0-9]\+\.[0-9]\+\.[0-9]\+$"; then
```

Comment:
The problem discussed in #173 is that for deployment we used Docker tags named after branches instead of tags. We need a clear way of distinguishing the two.

It seems like @scjody and you use the convention that branches do not have a PATCH version in their names but tags do. Is that really the case?

Reply:
I don't understand what you mean by "a PATCH version".

The convention used by the code managed by the devops team is:
* Release branches are of the form `3.0-release`, where `3.0` is the major.minor version of the release. (We don't follow semver but that's not relevant here - the convention works even if semver is used.)
* We add annotated tags to this branch to represent anything we release to customers.
* We add tags to Docker images using Quay's interface as part of adding them to an On-Prem release.

Feel free to adopt any of this that's useful to you and @etpinard, or come up with your own conventions provided they meet the following requirements:
* You're able to give us an Orca release using a Docker image tag that points to an unchanging image.
* Any Orca release that forms part of an On-Prem release can be minimally updated as part of a maintenance release. In other words we can address minor issues without changing any other parts of the code or any underlying dependencies.